app/src/main/java/com/stevesoltys/seedvault/transport/ConfigurableBackupTransport.kt

```diff
@@ -117,9 +122,9 @@ class ConfigurableBackupTransport internal constructor(private val context: Cont
 
     override fun isAppEligibleForBackup(
         targetPackage: PackageInfo,
-        isFullBackup: Boolean,
+        @Suppress("UNUSED_PARAMETER") isFullBackup: Boolean,
     ): Boolean {
-        return backupCoordinator.isAppEligibleForBackup(targetPackage, isFullBackup)
+        return settingsManager.isAppAllowedForBackup(targetPackage.packageName)
```

Comment:
It was actually intentional that everything from here passes through `backupCoordinator`. Could still leave it like this and just hand over to settingsManager there, then you don't need to inject it here.